src/routes/safe/components/AddressBook/ImportEntriesModal/index.tsx

```diff
@@ -63,8 +69,7 @@ const ImportEntryModal = ({ importEntryModalHandler, isOpen, onClose }) => {
     }
 
     const formatedList = slicedData.map((entry) => {
-      const address = entry.data[0]
-      return { address: checksumAddress(address), name: entry.data[1] }
+      return { address: checksumAddress(entry.data[0]), name: entry.data[1], chainId: entry.data[2] }
```

Comment:
I know I'm nitpicking here, but can we convert `entry.data[2]` to number?

`AddressBookEntry['chainId']` is `number`, and as I understand `entry.data[2]` is a `string`, am I right?

Reply:
@juampibermani 👆

Reply:
Sure